lib/disk/modules/VMWareSparseDisk.rb

```diff
@@ -2,49 +2,117 @@
 require 'disk/modules/miq_disk_common'
 require 'binary_struct'
 require 'memory_buffer'
+require 'zlib'
 
 module VMWareSparseDisk
+  SPARSE_MAGIC_NUMBER = 0x564d444b # 'V' 'M' 'D' 'K'
```

Comment:
Alternately,


```ruby
SPARSE_MAGIC_NUMBER = "VMDK".unpack("L>")[0]
```

Reply:
That looks much cleaner!  Maybe even `unpack1` instead of `unpack(‘L>‘)[0]`

Reply:
I had no idea `unpack1` was a thing but that is WAY nicer.... `"VMDK".unpack1("L>")`